app/models/entity.rb

```diff
@@ -4,6 +4,7 @@
 require 'hierarchical_relationships'
 require 'entity_index_queuer'
 require 'default_name_original'
+require 'elasticsearch/model'
```

Comment:
You shouldn't need this as `include Elasticsearch::Model` is already part of the `Entity` class.